src/main/java/com/beantastic/api/models/entities/BeanClassStatistic.java

```diff
@@ -0,0 +1,22 @@
+package com.beantastic.api.models.entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Data
+@Entity
+@Table
+public class BeanClassStatistic {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int beanClassStatisticsId;
+
+    @ManyToOne
+    private BeanClass beanClass;
```

Comment:
I think it makes sense to have this relationship be on the beanclass side

Reply:
still seeing it here